lib/faraday/adapter/net_http.rb

```diff
@@ -75,9 +75,43 @@ def create_request(env)
       end
 
       def perform_request(http, env)
+        if env[:request].stream_response?
+          size = 0
+          yielded = false
+          http_response = perform_request_with_wrapped_block(http, env) do |chunk|
+            if chunk.size > 0 || size > 0
+              yielded = true
+              size += chunk.bytesize
+              env[:request].on_data.call(chunk, size)
```

Comment:
In `README.md` it says the call's 2nd argument is chunk_size, but `size` here is total size.
Which does you intend?